in_toto/models/layout.py

```diff
@@ -47,8 +47,8 @@
 # import validators
 import in_toto.models.common as models__common
 import in_toto.models.link as models__link
-@attr.s(repr=False, init=False)
-class Layout(models__common.Signable):
+
+class Layout(models__common.Metablock):
```

Comment:
Doesn't below docstring belong to `_Layout` now? I'd rather explain here how and why `Layout` holds a `_Layout` in its `signed` property (inherited from the `Metablock` base class).